src/hpc_libs/interfaces/base.py

```diff
@@ -88,7 +88,7 @@ class Interface(ops.Object):
           a requirer/provider to successfully process a `RelationChangedEvent`.
     """
 
-    # A temporary fix for:
+    # A workaround for:
```

Comment:
[nit]: can the Launchpad URL be on the same line here? Having this statement on one line with the Launchpad URL is less than 99 characters:

```suggestion
    # A workaround for: https://bugs.launchpad.net/juju/+bug/1979811
```